server/settings/base.conf

```diff
@@ -0,0 +1,3 @@
+COMPOSE_FILE=docker-compose.yml

```

Comment:
we don't need this file, docker is prod only, and run.sh has this already.

Reply:
Removed.